src/types/BeachballOptions.ts

```diff
@@ -127,6 +128,11 @@ export interface RepoOptions {
   path: string;
   /** Prerelease prefix for packages that are specified to receive a prerelease bump */
   prereleasePrefix?: string | null;
+  /** This is for prerelease. Set it to "0" for zero-based or "1" for one-based.
+   *  Set it to false to omit the prerelease number.
+   *  @default "0"
+   */
+  identifierBase?: inc.IdentifierBase | false;
```

Comment:
To avoid a type reference to semver, can you change this to `'0' | '1' | false`? The type reference might not resolve in consumer repos if they're using types in their beachball config.

Reply:
Makes sense. Done!